### PR TITLE
Bootstrap master

### PR DESCRIPTION
**Pull request by lyingparachute**
Added bootstrap dependencies and configured spring security to enable unauthenticated user get access to bootstrap files.
New navigation bar and register page made with bootstrap.

### CODE REVIEW

File: myFit-webApp/pom.xml [hunk 1]
```diff
@@ -57,6 +57,41 @@
...
+        <dependency>
+            <groupId>org.webjars.bower</groupId>
+            <artifactId>jquery</artifactId>
+            <version>3.6.0</version>
+        </dependency>
+
+        <!-- https://mvnrepository.com/artifact/org.webjars/popper.js -->
+        <dependency>
+            <groupId>org.webjars</groupId>
+            <artifactId>popper.js</artifactId>
```
Comment (miosz) [inline myFit-webApp/pom.xml:78]: do we need all of those?

File: myFit-webApp/src/main/resources/templates/fragments/footer.html [hunk 2]
```diff
@@ -52,7 +43,9 @@ <h5 class="text-uppercase">Contact</h5>
         <a class="text-dark" href="https://infoshareacademy.com/">InfoShareAcademy.com</a>
     </div>
     <!-- Copyright -->
+    <script th:src="@{/webjars/jquery/3.6.0/dist/jquery.min.js}"></script>
```
Comment (miosz) [inline myFit-webApp/src/main/resources/templates/fragments/footer.html]: why in footer?

File: myFit-webApp/src/main/resources/templates/index.html [hunk 3]
```diff
@@ -1,22 +1,20 @@
...
 
 <div th:if="${param.logout}" class="card text-center">
     You have been logged out.
 </div>
 
-<div th:insert="~{fragments/footer :: footer}"></div>
-
+<div th:replace="fragments/footer :: footer"></div>
+<!--
+<script th:src="@{/webjars/jquery/3.6.0/dist/jquery.min.js}"></script>
```
Comment (miosz) [inline myFit-webApp/src/main/resources/templates/index.html]: could be deleted?

File: myFit-webApp/src/main/resources/templates/login.html [hunk 4]
```diff
@@ -11,12 +9,63 @@
...
 <form th:action="@{/login}" method="post">
     <div><label> User Name : <input type="text" name="username"/> </label></div>
     <div><label> Password: <input type="password" name="password"/> </label></div>
     <div><input type="submit" value="Sign In"/></div>
 </form>
-
+-->
 <div th:insert="~{fragments/footer :: footer}"></div>
+<script th:src="@{/webjars/font-awesome/5.15.4/css/all.min.css}"></script>
+<script th:src="@{/webjars/jquery/3.6.0/dist/jquery.min.js}"></script>
```
Comment (miosz) [inline myFit-webApp/src/main/resources/templates/login.html]: isn't it already in the footer fragments?

**Review by miosz — APPROVED** [review]

File: myFit-webApp/src/main/resources/templates/fragments/footer.html (hunk 2 above)
Comment (lyingparachute) [reply myFit-webApp/src/main/resources/templates/fragments/footer.html:46]: If I put it in files - all files need script. I put it in footer so 4 lines of code with script is only in footer

File: myFit-webApp/pom.xml (hunk 1 above)
Comment (lyingparachute) [reply myFit-webApp/pom.xml:78]: Yes, we do.  If we have all those, we don't need to download files and put them into resources

File: myFit-webApp/src/main/resources/templates/login.html (hunk 4 above)
Comment (lyingparachute) [reply myFit-webApp/src/main/resources/templates/login.html:102]: Yes, fixed it

File: myFit-webApp/src/main/resources/templates/index.html (hunk 3 above)
Comment (lyingparachute) [reply myFit-webApp/src/main/resources/templates/index.html:15]: Ofc

**Review by mateuszostrowski — APPROVED** [review]

**Pull request merged**